libmamba/src/core/package_handling.cpp

```diff
@@ -461,6 +461,13 @@ namespace mamba
         }
     }
 
+    fs::path extract_subproc(const fs::path& file)
```

Comment:
Dead code?

Reply:
Yup, this is part of my less-mergable follow-up PR and got caught in the wrong commit. I'll amend the commit tomorrow.